platform-dom/src/test/java/com/softicar/platform/dom/elements/input/auto/DomAutoCompletePopupTest.java

```diff
@@ -0,0 +1,73 @@
+package com.softicar.platform.dom.elements.input.auto;
+
+import com.softicar.platform.dom.DomI18n;
+import com.softicar.platform.dom.elements.testing.engine.IDomTestExecutionEngine;
+import com.softicar.platform.dom.elements.testing.engine.IDomTestExecutionEngineMethods;
+import com.softicar.platform.dom.elements.testing.engine.document.DomDocumentTestExecutionEngine;
+import com.softicar.platform.dom.elements.testing.node.tester.DomNodeTester;
+import com.softicar.platform.dom.event.DomEventType;
+import com.softicar.platform.dom.input.DomTextInput;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class DomAutoCompletePopupTest extends AbstractDomAutoCompleteDefaultInputEngineTest implements IDomTestExecutionEngineMethods {
```

Comment:
Hm, do I even need this test?
Wrote this before knowing all the Ajax tests

Reply:
not sure, we should ask Alex